hack/table-gen/table-gen.go

```diff
@@ -0,0 +1,264 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"sort"
+	"strings"
+
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	REPattern      = `(?s)<!--\s*TABLE-START\s* -->.*<!--\s*TABLE-END\s*-->` //Regular expression patter for reading everything between TABLE-START and TABLE-END tags
```

Comment:
```suggestion
	// Regular expression pattern for reading everything between TABLE-START and TABLE-END tags
	REPattern      = `(?s)<!--\s*TABLE-START\s* -->.*<!--\s*TABLE-END\s*-->` 
```

Reply:
It is more readable to have the comment above the variable to avoid scrolling to the right in order to read the comment